packages/loader/container-loader/src/loader.ts

```diff
@@ -202,10 +203,60 @@ export interface ILoaderProps {
 	readonly protocolHandlerBuilder?: ProtocolHandlerBuilder;
 }
 
+/**
+ * Other props which are used to load a container on top of the basic loader props.
+ * @alpha
+ * @legacy
```

Comment:
Consider just shipping as `@alpha` (not `@legacy`) until we've experimented with customer integration and have full confidence that we've made all the tweaks we want.

Reply:
+1. Are there any scenarios where we should be shipping new `@legacy` APIs? I would have guessed not.

Reply:
I suspect it is because we are hoping current `@legacy` customers will use it in the short-term but haven't made plans to get it to `@public` quality yet.  But even in that case I think it's wise to progress through at least a short `@alpha`-only phase to experiment and get feedback, maybe try some alternatives before publishing under `@legacy`.